tests/common.c

```diff
@@ -713,8 +708,10 @@ parsec_context_t* setup_parsec(int argc, char **argv, int *iparam)
                                    NULL);
         assert(-1 != CuHI);
     }
+    iparam[IPARAM_NGPUS] = nbgpu;
 #endif
 
+    print_arguments(iparam);
```

Comment:
This now prints the parameters the test runs with and not the parameters it got.

Reply:
That was the main source of astonishment. The output reports we used 0 gpu, then we go on and use as many as available on the platform.